apps/voting/app/src/components/LocalIdentityBadge/LocalIdentityBadge.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Badge, IdentityBadge, font } from '@aragon/ui'
+import { IdentityContext } from '../IdentityManager/IdentityManager'
+
+function useIdentity(address) {
```

Comment:
Should we move this to its own file so it's easier to share?

This also seems like could be included with the `IdentityManager`.

Reply:
This is the only place where it is used and eventually we'll move this to the `aragonAPI` so I don't think it is necessary to create a new file right now.